pages/linux/apx.md

```diff
@@ -1,17 +1,21 @@
 # apx
 
 > Package management utility for Vanilla OS.
-> Install packages inside managed containers from multiple sources (`apx` supports --aur,--dnf flags in all commands).
+> Install packages inside managed containers from multiple sources (`apx` supports --aur,--dnf, --apk flags in all commands).
 > More information: <https://github.com/Vanilla-OS/apx>.
 
-- Initialize the container:
+- Initialize/Reinitialize the container:
 
 `apx init`
 
 - Install specific packages in the container:
 
 `apx install {{package1 package2 ...}}`
 
+- Install a DEB/RPM package inside the container (Use `--dnf` flag for installing RPMs):
+
+`apx install --sideload {{path/to/package{{.deb|.rpm}}}}`
```

Comment:
```suggestion
- Install a DEB/RPM package inside the container (Use `--dnf` flag for installing RPMs):

`apx install --sideload {{path/to/package.ext}}`
```

Mandatory fix: unexpected `path/to/package{{.deb|.rpm}}` placeholder content, `ext` expected.
Note: nested placeholders are not supported in specification.